### PR TITLE
Eynollah light integration

### PR DESCRIPTION
**Pull request by cneud**
We should start integration of `eynollah_light` with the `main` branch - this helps track what conflicts etc. need to be resolved.

### CODE REVIEW

**Review by vahidrezanezhad — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -33,28 +33,29 @@ In case you want to train your own model to use with Eynollah, have a look at [s
...
--ep <if true, the tool will be enabled to save desired plot. This should be true alongside with -sl, -sd, -sa , -si or -ae options>
--light <if true, the tool will apply a faster method for main regions detection and moreover deskewing will not occur locally for each region in return it is done once for the whole document>
--di <directory of images. This option will accelerate the process significantly>
+eynollah -i <image file name> -o <directory to write output> -m <directory of models> [OPTIONS]
+```
+
+Additionally, the following optional parameters can be used to further configure the processing:
 
+```sh
+-fl: the tool will perform full layout analysis including detection of marginalia and drop capitals
```
Comment (vahidrezanezhad) [inline README.md]: full layout detection is responsible for header and drop capitals not marginalia.

File: README.md [hunk 2]
```diff
@@ -38,21 +38,21 @@ eynollah -i <image file name> -o <directory to write output> -m <directory of mo
...
--sa <directory>: save plot of layout detection to this directory  
--tab: try to detect tables  
--ib: allow binarization of the input image  
--ho: ignore headers in reading order detection  
--sl <directory>: save plots of layout detection to this directory  
--ep: save a plot. This should be used alongside with `-sl`, `-sd`, `-sa`, `-si` or `-ae` options  
--light: apply a faster but simpler method for main region detection and deskewing   
--di <directory>: process all images in a directory in batch mode
+```
+-fl              perform full layout analysis including detection of marginalia and drop capitals
```
Comment (vahidrezanezhad) [inline README.md:42]: perform full layout analysis including detection of headers and drop capitals

**Comment by cneud** [comment]
Not sure why CI fails, as it works locally, possibly also https://github.com/qurator-spk/eynollah/issues/98?

**Comment by cneud** [comment]
@vahidrezanezhad: I've started backporting/cherry-picking and testing most of the recent changes from [main](https://github.com/qurator-spk/eynollah/commits/main) into [eynollah_light](https://github.com/qurator-spk/eynollah/commits/eynollah_light).

I did not yet do this for two changes that I would like you to check and adapt/integrate with `eynollah_light`:

1) [remove TF1 session and GC controls, avoid repeating load_model](https://github.com/qurator-spk/eynollah/commit/7345f6bf678f36cf3a51576b0fa94df0919925d7)

2) [contours: simplify ](https://github.com/qurator-spk/eynollah/commit/2d9ccac35416afe24553147c429035c94cc2bf24) (which implies [contours: numpy now needs dtype=object ](https://github.com/qurator-spk/eynollah/commit/a56988a35a528aba7cefd85e1a83257f598a5085))

Once this is done, we should be able to resolve any remaining conflicts, which would give us a good basis for merging `eynollah_light` into `main`.

**Comment by cneud** [comment]
With 456fccb35e184db1e7a0e73965e446ed8993f41a the CircleCI cache got updated but now both jobs get killed (due to RAM exhaustion?) during the smoke-test 🌧️ GitHub actions still fails with the old cache 🌩️ 

Locally I get the a working installation with Python 3.7./TF2.11 or Python 3.8/TF2.12 and the `2022-04-05` models in `SavedModel` format.

**Review by vahidrezanezhad — APPROVED** [review]

**Pull request merged**